content/en/security/threat_intelligence.md

```diff
@@ -66,7 +66,7 @@ Sources, categories, and intents are available as facets and filters on relevant
 
 | Source | Category | Source Use Cases | Primary Products | 
 |--------|------------|-----------|------------------|
-| Datadog Threat Research| scanners, exploits | Honeypots focused on software specific threats | ASM and CWS |
+| Datadog Threat Research| scanners, redis exploitation, docker exploitation, malware, bruteforcer | Honeypots focused on software specific threats | ASM, CWS and Cloud SIEM |
```

Comment:
Are these tags of some sort, or should these be using proper capitalization of e.g. Redis and Docker?

Reply:
These are types.

Reply:
Okay great, thanks for clarifying for me